jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/AbstractBaseEndpoint.java

```diff
@@ -73,12 +73,12 @@ protected void validateModelClass(Object model) {
      * @return 400 Bad Request response, containing details on the constraint violations 
      */
     protected ResponseBuilder createBadRequestResponse(Set<ConstraintViolation<?>> violations) {
-        final Map<String, String> responseObj = new HashMap<String, String>();
+        final Map<String, String> responseObj = new HashMap<>();
 
-        for (ConstraintViolation<?> violation : violations) {
+        violations.forEach(violation -> {
             responseObj.put(violation.getPropertyPath().toString(),
-                                violation.getMessage());
-        }
+                    violation.getMessage());
+            });
```

Comment:
@idelpivnitskiy good suggestions. Mind sending a PR against my branch ? :)

On Tue, May 24, 2016 at 12:44 AM, Idel Pivnitskiy <notifications@github.com>
wrote:

> In
> jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/AbstractBaseEndpoint.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/721#discussion_r64302310>
> :
>
> >              responseObj.put(violation.getPropertyPath().toString(),
> > -                                violation.getMessage());
> > -        }
> > +                    violation.getMessage());
> > +            });
>
> Maybe:
>
> final Map<String, String> responseObj = violations.stream().collect(Collectors.toMap(
>         v -> v.getPropertyPath().toString(), ConstraintViolation::getMessage));
>
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/721/files/08f47991352b54c61a71a8f4f2b8efc4e6cf5de1#r64302310>
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
twitter: http://twitter.com/mwessendorf